app/Http/ViewComposers/IngameMainComposer.php

```diff
@@ -102,13 +107,20 @@ public function compose(View $view): void
         // Get current locale
         $locale = App::getLocale();
 
+        $playerId = (string) $this->player->getId();
+
+        $highscoreRank = Cache::remember($playerId, now()->addMinutes(5), function () {
```

Comment:
Could do with changing this to `'player-highscore-'.$playerId` - then might not need to cast the playerId to string.